src/main/scala/firrtl/util/BackendCompilationUtilities.scala

```diff
@@ -107,7 +107,8 @@ trait BackendCompilationUtilities extends LazyLogging {
     vSources: Seq[File],
     cppHarness: File,
     suppressVcd: Boolean = false,
-    resourceFileName: String = firrtl.transforms.BlackBoxSourceHelper.defaultFileListName
+    resourceFileName: String = firrtl.transforms.BlackBoxSourceHelper.defaultFileListName,
+    extraCmdLineArgs: Seq[String] = Seq.empty
```

Comment:
This is break binary compatiblity with the 1.3.X branch, which means it can't be backported. Instead, create a new function with the same name and the additional optional argument, call that function from this function, and deprecate this function.

Reply:
I'm a little late here, but I'm not a huge fan of deprecating `verilogToCpp` in favor of a very long name `verilogToCppWithExtraCmdLineArgs`, can we either:
1) Not deprecate `verilogToCpp`
2) Have `verilogToCppWithExtraCmdLineArgs` be named `verilogToCpp` instead and just overload?

Reply:
This was the initial approach, but it turns out you can't overload methods with default arguments :(

Reply:
i'm not really sure how deprecation works, but is it possible that when `verilogToCpp` is removed, to rename `verilogToCppWithExtraCmdLineArgs` to `verilogToCpp`?

Reply:
We could just not deprecate `verilogToCpp`, it's not that big of a deal to have two functions

Reply:
That sounds reasonable to me. Any opinions, @azidar ?